code/game/objects/structures/skele_stand.dm

```diff
@@ -0,0 +1,38 @@
+/obj/structure/skele_stand
+	name = "hanging skeleton model"
+	anchored = 1
+	density = 1
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "hangskele"
+	desc = "It's an anatomical model of a human skeletal system made of plaster."
+	var/last_rattle_time = 0
+	var/rattle_cooldown = 2 SECONDS// let's not get our rattling out of hand
+
+/obj/structure/skele_stand/New()
+	..()
+	gender = pick(MALE, FEMALE)
```

Comment:
I think it may affect usage of the `\he`, `\him`, etc macros? Also maybe the examine text.